crates/stages/stages/src/stages/execution.rs

```diff
@@ -404,6 +434,17 @@ where
 
         Ok(UnwindOutput { checkpoint })
     }
+
+    fn post_unwind_commit(&mut self) -> Result<(), StageError> {
+        let Some(chain) = self.post_unwind_commit_input.take() else { return Ok(()) };
+
+        // NOTE: We can ignore the error here, since an error means that the channel is closed,
+        // which means the manager has died, which then in turn means the node is shutting down.
+        let _ =
+            self.exex_manager_handle.send(ExExNotification::ChainReverted { old: Arc::new(chain) });
```

Comment:
can we add a function fn send_chain(Chain), because we have this twice

Reply:
but we send different types of notifications: one is commit and the other is revert

Reply:
or do you mean `send_chain_committed`, `send_chain_reverted`, etc?

Reply:
ah gotcha